src/app/app.component.html

```diff
@@ -8,9 +8,30 @@ <h2>Altrusight</h2>
         <span class="spacer"></span>
         <div class="centered-content">
           <mat-form-field class="search-bar" color="primary" appearance="outline">
+
+
+            <!-- Search Nonprofits input -->
             <mat-label>Search Nonprofits</mat-label>
-            <input matInput placeholder="Search">
-            <mat-icon matPrefix>search</mat-icon>
+            <!-- KEEP CODE BELOW: In case we want to improve search funtinality -->
```

Comment:
Let's readd the dropdown if possible. I think it would be faster if you just took a quick filter of the top 5. I think why it might be so slow is that the performance is pretty low with more than a few items, just a guess though

Reply:
For now, the dropdown is not totally a great idea because the API does not return the most relevant search results at the top. Meaning the user would have to scroll for a while before finding their desired results. I did try to keep it but it was kind of complicated to make it dynamic to where the results adjust as the user types so that's another reason why I went with the current implementation. Since the semester is almost over and my time is already pretty thin with other classes, I think the current functionality would be good. But I think the nonprofits search that displays the results as a name only could also include like city/ state/ EIN to show that results with the same name are not the same nonprofit.